lazyscribe/project.py

```diff
@@ -90,6 +90,11 @@ def load(self):
         If the project is in read-only or append mode, existing experiments will
         be loaded in read-only mode. If opened in editable mode, existing experiments
         will be loaded in editable mode.
+
+        Raises
+        ------
+        SaveError
+            Raised when writing to the filesystem fails.

```

Comment:
Are you sure this belongs to the `.load()` method?

Reply:
Moved. Thanks.